templates/include/glean/event.1.parquetmr.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "array",
+  "items": [
+    {
+      "type": "integer",
+      "minimum": 0
+    },
+    {
+      "type": "string"
```

Comment:
Maybe add `"maxLength": 80` here to match the Kotlin code?

Reply:
Both `value` (which had the 80 chars limit) and `object` where dropped. This here is the category name: I'm not sure we have an hardwire limit in kotlin for that

Reply:
Oh, good point.  No limit on this one (for now).